MediaBrowser.Model/Entities/EncoderPreset.cs

```diff
@@ -0,0 +1,59 @@
+#pragma warning disable SA1300 // Lowercase required for backwards compat.
+
+namespace MediaBrowser.Model.Entities;
+
+/// <summary>
+/// Enum containing encoder presets.
+/// </summary>
+public enum EncoderPreset
+{
```

Comment:
We do have an "auto" preset which is basically an empty string so that it will be assigned to the per-encoder default. Now this breaks all auto cases

Reply:
Where is this used?

Reply:
When we map the preset to encoder specific qualities, we do have a case that is not any of those value which is the real default "auto" value